libraries/AP_BattMonitor/AP_BattMonitor_BLHeliESC.cpp

```diff
@@ -15,9 +15,9 @@
 
 
 #include <AP_HAL/AP_HAL.h>
```

Comment:
best to rename to AP_BattMonitor_ESC.cpp ?
and change param doc strings

Reply:
Fixed